Assets/Scripts/PlayerController.cs

```diff
@@ -44,10 +46,10 @@ private void Update()
         h = Input.GetAxisRaw("Horizontal");

```

Comment:
I think it will be clearer to make everything that's in Update() here into its own method with a summary